Flipper/Packages/UI/Sources/Archive/Info/View/BaseInfoView.swift

```diff
@@ -31,7 +32,12 @@ struct BaseInfoView: View {
                             title: "Edit Dump",
                             action: onOpenNFCEdit
                         )
-                        .foregroundColor(.primary)
+                        .foregroundColor(
+                            isEmulating
```

Comment:
Would be better to extract the button to it's own view and use isEnabled to choose the color